feat: add treasury proposal page from the homepage to the docs (#96)
KILTprotocol/docs: docs/participate/treasury-proposal.md

---
id: treasury-proposal
title: Create a treasury proposal
---

Complete these steps to create a well-formed Treasury proposal

# Discuss

> The first step in applying for a Treasury grant is to join the community in the [Element Direction Channel](https://matrix.to/#/#kilt-direction:matrix.org), [Discord Treasury Channel](https://discord.gg/nUpqDfQ6kJ) or on [Telegram](https://t.me/KILTProtocolChat) to discuss and share your intended proposal. This will help you get valuable community feedback throughout the process. This also gives Council members an open and transparent way to measure community sentiment.  

Reviewer: 1. Unfortunately comms have not updated the site as requested (and approved) 2022-05-10. Instead of/additional to channeling folks to Element, Discord, or Telegram, we wanted to add [Polkassembly Discussion](https://kilt.polkassembly.network/discussions) to the list. We devs and Council would like to have all "formal" discussions on Polkassembly. Discord should only serve as an option to "brainstorm". However, once they have a proposal draft, we want it to be posted to Polkassembly
2. New line please
```suggestion
The first step in applying for a Treasury grant is either to join the community in the [Discord Treasury Channel](https://discord.gg/nUpqDfQ6kJ) to brainstorm the scope of your proposal intention or immediately open a discussion on [Polkassembly](https://kilt.polkassembly.network/discussions). 
This will help you get valuable community feedback throughout the process.
It also gives Council members an open and transparent way to measure community sentiment.  
```

# Deposit

> A deposit of 5% of the amount requested is required in order to submit a proposal. If the proposal is denied, you will lose this amount and it will go to the Treasury to fund other projects. This is why it is essential to engage with the community and show how the proposed work adds value to the network. 

Reviewer: New lines

```suggestion
A deposit of 5% of the amount requested is required in order to submit a proposal.
If the proposal is denied, you will lose this amount and it will go to the Treasury to fund other projects.
This is why it is essential to engage with the community and show how the proposed work adds value to the network. 
```

# Proposal Document

> To maximize your chances of success, create a full proposal document with as much information as possible to communicate the value of your work and what it will add to the network’s growth and success. Check out our Treasury proposal template or Polkassembly’s [proposal document #6](https://docs.google.com/document/d/1NilDtFljCc6boyHkz2o2kGyFNhR6hVQSGzz7J0Xhwco/edit) as examples to help guide your process.

Reviewer: New line
```suggestion
To maximize your chances of success, create a full proposal document with as much information as possible to communicate the value of your work and what it will add to the network’s growth and success.
Check out our Treasury proposal template or Polkassembly’s [proposal document #6](https://docs.google.com/document/d/1NilDtFljCc6boyHkz2o2kGyFNhR6hVQSGzz7J0Xhwco/edit) as examples to help guide your process.
```

> Once your proposal document is complete, upload it so that it’s accessible to the Council for review. Don’t forget to link it when completing the Polkassembly information! 

Reviewer: New line

```suggestion
Once your proposal document is complete, upload it so that it’s accessible to the Council for review.
Don’t forget to link it when completing the Polkassembly information! 
```

# Submit Proposal

When you have feedback from the community and are satisfied with your proposal, head to [polkadot apps](https://polkadot.js.org/apps/?rpc=wss://spiritnet.api.onfinality.io/public-ws#/treasury)

Scroll to  the “+ Submit Proposal” button and click.
Complete the form:

Submit with Account: this is the account that will make the 5% deposit 
Beneficiary: this is the account that will receive Treasury funding if successful 
Value: this is the full amount of KILT being requested

> Click submit to complete the proposal.

Reviewer: Please add the missing bullet points
```suggestion
* Submit with Account: this is the account that will make the 5% deposit 
* Beneficiary: this is the account that will receive Treasury funding if successful 
* Value: this is the full amount of KILT being requested
* Click submit to complete the proposal.
```

# Polkassembly Details

> Click the Polkassembly link to the right of the proposal on Polkadot-JS

Reviewer: C: Add link to Polkassembly here

```suggestion
Click the [Polkassembly](https://kilt.polkassembly.network/discussions) link to the right of the proposal on Polkadot-JS
```

Enter the following information:

* Title: a title for your project
* Purpose: what’s your motivation behind the effort
* Description: a short summary about you, the project and the need for what you are proposing.
* Full Proposal: link to the full proposal
* Proposal Amount: the amount requested in USD
* KILT Rate: the current rate of exchange in USD
* Amount Requested: the total amount of KILT requested
* Beneficiary: the beneficiary address
* Period: intended date of delivery if applicable
* Contact: email or social handle (please specify social network)
* Engage
* Share your proposal in our channels to generate support from community and showcase your project (Discord, Telegram, Element, Twitter).

> Click [here](https://www.kilt.io/treasury/overview/) to get an overview about Treasury. Or click [here](https://www.kilt.io/treasury/content-creation/) to see the guidelines for content creation.

Reviewer: New line
```suggestion
Click [here](https://www.kilt.io/treasury/overview/) to get an overview about Treasury.
Or click [here](https://www.kilt.io/treasury/content-creation/) to see the guidelines for content creation.
```

# Proposal Document

To maximize your chances of success, create a full proposal document with as much information as possible to communicate the value of your work and what it will add to the network’s growth and success.
Check out our Treasury proposal template or Polkassembly’s [proposal document #6](https://docs.google.com/document/d/1NilDtFljCc6boyHkz2o2kGyFNhR6hVQSGzz7J0Xhwco/edit) as examples to help guide your process.

Once your proposal document is complete, upload it so that it’s accessible to the Council for review.
Don’t forget to link it when completing the Polkassembly information! 

# Submit Proposal

When you have feedback from the community and are satisfied with your proposal, head to [polkadot apps](https://polkadot.js.org/apps/?rpc=wss://spiritnet.api.onfinality.io/public-ws#/treasury)

Scroll to  the “+ Submit Proposal” button and click.
Complete the form:

* Submit with Account: this is the account that will make the 5% deposit 
* Beneficiary: this is the account that will receive Treasury funding if successful 
* Value: this is the full amount of KILT being requested
* Click submit to complete the proposal

# Polkassembly Details

> Click the [Polkassembly](https://kilt.polkassembly.network/discussions) link to the right of the proposal on Polkadot-JS

Reviewer 2: I guess we should mention that the user needs to login with the account that opened the proposal in order to modify it.
